project_liberation/main_page/templates/main_page/footer.haml

```diff
@@ -1,2 +1,51 @@
+                %section.widget.footer_address
+                    %h3.widgettitle Code Poets sp. z o.o.
+                    .textwidget.custom-html-widget
+                        ul. Józefa Piłsudskiego 49-57
+                        %br
+                        50-032 Wrocław
+                        %br
+                        Poland
+                        %br
+                        %br
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
+                %section.widget.footer_contact
+                    %h3.widgettitle Contact Us
+                    .textwidget.custom-html-widget
+                        .anti-bot-style ti.steopedoc@tcatnoc
+                %section.widget.footer_contact-2
+                    .textwidget
+                        .social-media-container
+                            %a{:href => "https://www.facebook.com/codepoets.it/"}
+                                %img{:src => "/static/main_page/images/footer/social_icons/facebook_icon.png", :alt => "Facebook", :target => "_blank", :width => "25px", :height =>"25px"}
+                            %a{:href => "https://www.instagram.com/codepoets.it/"}
+                                %img{:src => "/static/main_page/images/footer/social_icons/instagram_icon.png", :alt => "Instagram", :target => "_blank", :width => "25px", :height =>"25px"}
+                            %a{:href => "https://www.linkedin.com/company/codepoets/"}
+                                %img{:src => "/static/main_page/images/footer/social_icons/linkedin_icon.png", :alt => "LinkedIn", :target => "_blank", :width => "25px", :height =>"25px"}
+                            %a{:href => "https://clutch.co/profile/code-poets/"}
+                                %img{:src => "/static/main_page/images/footer/social_icons/clutch_icon.png", :alt => "Clutch", :target => "_blank", :width => "25px", :height =>"25px"}
+                            %a{:href => "https://github.com/code-poets/"}
+                                %img{:src => "/static/main_page/images/footer/social_icons/github_icon.png", :alt => "GitHub", :target => "_blank", :width => "25px", :height =>"25px"}
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
+                %section.widget.footer_reviews
+                    .textwidget
+                        %script{:type => "text/javascript", :src => "https://widget.clutch.co/static/js/widget.js"}
+                        .clutch-widget{:data-url => "https://widget.clutch.co", :data-widget-type => "2", :data-height => "100", :data-darkbg => "1", :data-clutchcompany-id => "705656"}
+                            %iframe#iframe-0.04822719047585822{:frameborder => "0", :width => "100%", :height => "100px", :scrolling => "no", :src => "https://widget.clutch.co/widgets/get/2/darkbg?ref_domain=codepoets.it&amp;uid=705656", :style => "display: block;"}
```

Comment:
`:style => "display: block;"` add to sass please :)

Reply:
@maciejSamerdak HOW COULD YOU USE IN-LINE STYLE :O

Reply:
@kbeker DON'T YOU CAPITALISE AT ME, MISTER! >:'(

In all seriousness though, I pretty much copy-pasted this fella. Among all these Clutch-specific parameters I must've ignored the style attribute.

Although it does seems a little bit silly to me, although accurate, to create another class in sass just for a single line of code with `display` with all this other stuff in here...
Unless something else goes to sass as well? 😕

Reply:
Done